pom.xml

```diff
@@ -46,7 +46,7 @@
         <gson.fire.version>1.8.2</gson.fire.version>
         <gson.version>2.6.1</gson.version>
         <hibernate.validator.version>6.0.16.Final</hibernate.validator.version>
-        <immutables.version>2.7.1</immutables.version>
+        <immutables.version>2.8.0-CriteriaPreview0</immutables.version>
```

Comment:
This doesn't look like a version to which we want to bump, right?

Reply:
Nope, I think we can just chose to ignore the minor version with one of the bot commands and reopen this PR later.

Reply:
@jhunzik I think we should not ignore 2.8.x versions, only this specific version. So, we need to undo this command: https://github.com/connexta/ion-ingest-api/pull/28#issuecomment-519265452.